File: Lv.1/C,C++/소수 만들기.c
#include <stdio.h>
#include <stdbool.h>
#include <stdlib.h>

int solution(int nums[], size_t nums_len) {
    int answer = 0;
    int sum = 0;
    for (int i = 0; i < nums_len; i++) {
        for (int j = i + 1; j < nums_len; j++) {
            for (int k = j + 1; k < nums_len; k++) {
                sum = nums[i] + nums[j] + nums[k];
                int count = 0;
                for (int h = 2; h < sum; h++) {
                    if (sum % h == 0)
                        count++;
                }
                if (count == 0)
                    answer++;
            }
        }
    }
    return answer;
}

File: Lv.1/Python/소수 만들기.py
from itertools import combinations
def prime(n):
    for i in range(2,n):
        if n % i == 0:
            return False
    return True

def solution(nums):
    answer =0
    lst = list(combinations(nums,3))
    sum_ = []
    for i in lst:
        sum_.append(sum(i))
    for i in sum_:
        if prime(i):
            answer+=1
    return answer